crates/aptos/src/common/utils.rs

```diff
@@ -295,7 +296,7 @@ pub async fn fund_account(
     faucet_url: Url,
     num_coins: u64,
     address: AccountAddress,
-) -> CliTypedResult<()> {
+) -> CliTypedResult<Vec<HashValue>> {
```

Comment:
Is it possible to just wait on the final transaction?

Reply:
in the current implementation, yes. But that seems like an implementation detail. If transactions for something end up being issued from a different accounts, they can be reordered, and only one inserted in the block. 
So I'll leave as is.